fgpyo/util/metric.py

```diff
@@ -506,6 +508,7 @@ def __init__(
             f=self._fout,
             fieldnames=self._fieldnames,
             delimiter=delimiter,
+            **kwargs,
```

Comment:
We should probably change the default behavior of this class to override `DictWriter` and use `"\n"` as the default lineterminator.

Maybe check to see if `lineterminator` is in the user-specified `kwargs` and assign `"\n"` if not